src/System.Private.CoreLib/src/System/Runtime/CompilerServices/ConditionalWeakTable.cs

```diff
@@ -654,8 +636,8 @@ private void VerifyIntegrity()
 
             private int[] _buckets;             // _buckets[hashcode & _buckets.Length] contains index of first entry in bucket (-1 if empty)
             private Entry[] _entries;
-            private int _freeList;            // -1 = empty, else index of first unused Entry
-            private bool _invalid;             // flag detects if OOM or other background exception threw us out of the lock.
+            private int _firstFreeEntry;             // _firstFreeEntry < _entries.Length => table has capacity,  entries grow from the bottom of the table.
```

Comment:
Nit: Could you please fix the indentation of the comment to match the surrounding comments?